src/plugins/score-plugin-scenario/Scenario/Application/Menus/ObjectsActions/EventActions.cpp

```diff
@@ -154,7 +156,15 @@ void EventActions::addTriggerToTimeSync()
       **selectedTimeSyncs.begin());
 
   if (cmd)
+  {
     dispatcher().submit(cmd);
+    auto trigger = *selectedTimeSyncs.begin();
```

Comment:
since *selectedTimeSyncs.begin() is used at multiple places, could it be put in a small

     auto& selectedSync = *selectedTimeSyncs.begin(); 

?